pom.xml

```diff
@@ -75,17 +75,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.main</groupId>
-            <artifactId>jenkins-test-harness</artifactId>
-            <version>2109.v930e1e518c15</version>
-        </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>matrix-auth</artifactId>
-            <version>1.0.2</version>
-            <scope>test</scope>
-        </dependency>
```

Comment:
At least one test imports `hudson.security.GlobalMatrixAuthorizationStrategy` so I think this explicit test-scoped dependency should be kept. The version can be ommitted to use the one from the BOM if desired.

Reply:
Never mind, I didn't read carefully enough. This is a duplicate.